src/laboratory/utils_base_unit.py

```diff
@@ -0,0 +1,41 @@
+from laboratory.models import BaseUnitValues
+
+# from laboratory.utils_base_unit import get_conversion_units
+
+
+def get_base_unit(unit):
```

Comment:
You need to return something if query result does not exists. Where is applied this function??